### PR TITLE
Use AVA

### PR DESCRIPTION
**Pull request by arthurvr**

### CODE REVIEW

**Comment by kevva** [comment]
lgtm

File: test.js [hunk 1]
```diff
@@ -17,26 +16,26 @@ Object.keys(ansi).forEach(function (x) {
...
+test('should group related codes into categories', t => {
+	t.is(ansi.colors.magenta, ansi.magenta);
+	t.is(ansi.bgColors.bgYellow, ansi.bgYellow);
+	t.is(ansi.modifiers.bold, ansi.bold);
 });
 
-it('groups should not be enumerable', function () {
-	assert.equal(Object.keys(ansi).indexOf('modifiers'), -1);
+test('groups should not be enumerable', t => {
+	t.is(Object.keys(ansi).indexOf('modifiers'), -1);
```
Comment (arthurvr) [inline test.js]: @kevva WDYT about a helper method in `claim` for this kind of checks?

Comment (sindresorhus) [inline test.js]: IMHO too obscure, and all these kinds of checks will have much better debuggability when we add support for [power-assert](https://github.com/power-assert-js/power-assert).

File: test.js [hunk 2]
```diff
@@ -17,26 +16,26 @@ Object.keys(ansi).forEach(function (x) {
...
 
-it('should group related codes into categories', function () {
-	assert.equal(ansi.colors.magenta, ansi.magenta);
-	assert.equal(ansi.bgColors.bgYellow, ansi.bgYellow);
-	assert.equal(ansi.modifiers.bold, ansi.bold);
+test('should group related codes into categories', t => {
+	t.is(ansi.colors.magenta, ansi.magenta);
+	t.is(ansi.bgColors.bgYellow, ansi.bgYellow);
+	t.is(ansi.modifiers.bold, ansi.bold);
 });
```
Comment (sindresorhus) [inline test.js]: Tests need to end with `t.end();`

File: test.js (hunk 1 above)
Comment (shinnn) [inline test.js]: We can use [`Array#includes()`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) via AVA's built-in ES2015 support.

```js
t.true(Object.keys(ansi).includes('modifiers'));
```

File: test.js [hunk 3]
```diff
@@ -17,26 +16,30 @@ Object.keys(ansi).forEach(function (x) {
...
+	t.is(ansi.colors.magenta, ansi.magenta);
+	t.is(ansi.bgColors.bgYellow, ansi.bgYellow);
+	t.is(ansi.modifiers.bold, ansi.bold);
+	t.end();
 });
 
-it('groups should not be enumerable', function () {
-	assert.equal(Object.keys(ansi).indexOf('modifiers'), -1);
+test('groups should not be enumerable', t => {
+	t.false(Object.keys(ansi).includes('modifiers'));
```
Comment (Qix-) [inline test.js]: Why not `'modifiers' in ansi`?

File: test.js (hunk 1 above)
Comment (Qix-) [inline test.js]: Why not `t.false('modifiers' in ansi);`?

File: test.js (hunk 3 above)
Comment (Qix-) [inline test.js]: Still curious as to why this isn't `'modifiers' in ansi`

**Comment by sindresorhus** [comment]
To ensure to only check own keys. Feel free to change it.

**Comment by Qix-** [comment]
Ah okay.

**Pull request merged**